crmsh/log.py

```diff
@@ -24,7 +24,7 @@ class ConsoleCustomHandler(logging.StreamHandler):
     """
 
     def emit(self, record):
-        if record.levelno < logging.ERROR:
+        if record.levelno < logging.DEBUG:
```

Comment:
The int value of logging.DEBUG is the smallest. Are you going to send all logs to stderr?

Reply:
Thanks for reminding me! I forget that:)

Reply:
Already changed, only let INFO messages in stdout, others in stderr